source/agora/consensus/protocol/Nominator.d

```diff
@@ -91,17 +95,18 @@ extern(D):
             ledger = needed for SCP state restoration & block validation
             taskman = used to run timers
             peers = the set of clients to the peers in the quorum
-            quorum_set = the quorum set of this node
+            config = the quorum configuration
 
     ***************************************************************************/
 
-    public this (KeyPair key_pair, Ledger ledger,
-        TaskManager taskman, NetworkClient[PublicKey] peers,
-        SCPQuorumSet quorum_set)
+    public this (KeyPair key_pair, Ledger ledger, TaskManager taskman,
+        NetworkClient[PublicKey] peers, ref const QuorumConfig config)
     {
+        this.quorum_conf = config;
         this.key_pair = key_pair;
         auto node_id = NodeID(StellarHash(key_pair.address[]));
         const IsValidator = true;
+        auto quorum_set = verifyBuildSCPConfig(config);
```

Comment:
Pass it directly on the next line to avoid a copy.

Reply:
I can't, it takes a reference.